client/basic-authorship/src/basic_authorship.rs

```diff
@@ -371,6 +398,17 @@ mod tests {
 			txpool.submit_at(&BlockId::number(0), vec![extrinsic(0)]),
 		).unwrap();
 
+		futures::executor::block_on(
+			txpool.maintain(
```

Comment:
This is repeated couple of times already, can we extract it into a helper function?